snuba/datasets/configuration/discover/storages/discover.yaml

```diff
@@ -88,6 +88,7 @@ schema:
         args: { schema_modifiers: [nullable], size: 64 },
       },
       { name: deleted, type: UInt, args: { size: 8 } },
+      { name: trace_id, type: UUID, args: { schema_modifiers: [nullable] } }
```

Comment:
why are we adding trace_id? Isn't this pr a followup to the migration where you added replay_id?

Reply:
Do you also have to add a column to the entities/discover.yaml file?

Reply:
thanks for the catch, should have been replay_id